specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -1771,6 +1929,148 @@
           "type": "integer",
           "format": "int64"
         }
+      },
+      "required": [
+        "createTime"
+      ]
+    },
+    "AcsCallParticipantEventData": {
+      "description": "Schema of common properties of participant events",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsCallingEventProperties"
+        }
+      ],
+      "properties": {
+        "user": {
+          "description": "The communication identifier of the participant",
+          "$ref": "#/definitions/AcsCallParticipantUserProperties"
+        },
+        "displayName": {
+          "description": "The display name of the participant.",
+          "type": "string"
+        },
+        "participantId": {
+          "description": "The Id of the participant.",
+          "type": "string"
+        },
+        "userAgent": {
+          "description": "The user agent of the participant. ",
+          "type": "string"
+        }
+      }
+    },
+    "AcsCallingEventProperties": {
+      "description": "Schema of common properties of all calling events",
+      "type": "object",
+      "properties": {
+        "startedBy": {
+          "description": "The communication identifier of the call started by",
+          "$ref": "../../../../../communication/data-plane/Common/stable/2023-11-15/common.json#/definitions/CommunicationIdentifierModel"
```

Comment:
this should be -  "$ref": "#/definitions/AcsCallParticipantUserProperties"
![{2192BCC6-069F-49FF-9643-1769C2C9941D}](https://github.com/user-attachments/assets/7a65435b-8ad0-407a-8d2c-f79223ee4bb4)